File: week_02/src/main/java/Question09.java
//Autor : Zaid Nayeb
//Date  : 1404/08/26

//Grade Calculator (0–100 → A–F)
//Description: Input a student’s marks (0–100) and print the grade based on a scale.
//Example:
//Input: 85
// Output:B

public class Question09 {
    public static String gradeCalculator(double score){
        if (score < 60){
            return "F";
        }else if (score < 70){
            return "D";
        }else if (score < 80){
            return "C";
        }else if (score < 90){
            return "B";
        }else {
            return "A";
        }
    }
}

File: week_03/src/Question_09.js
// Author : Zaid Nayeb
// Date   : 2025/11/26
/*
9.	Write a function to count occurrences of an element in an array Description: Count how many times a given number appears in the array. Example: 
Input: 
Array = [2, 3, 2, 5, 2], Element = 2 Output: 
3 


*/
function countOccurrences(arr, element) {
  let count = 0;

  for (let i = 0; i < arr.length; i++) {
    if (arr[i] === element) {
      count++;
    }
  }

  return count;
}

console.log(countOccurrences([2, 3, 2, 5, 2], 2));
